### DISCUSSION TITLE
Is there a way to use deepgram live transcription with Asterisk PBX

### DISCUSSION THREAD

**Discussion by aqeelabpro** [deepgram/community#530] [General help]
I am new to Deepgram, I wanted to use Deepgram together with Asterisk PBX for real-time speech recognition. Is there a way to implement this with Asterisk PBX? I used https://www.teleconnx.com/dg-stt/get-started.html but it did not help me.

If it is possible can I use deepgram directly? as a paid membership?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
